nakama/nakama.lua

```diff
@@ -4634,25 +3098,34 @@ end
 --- write_tournament_record2
 -- Write a record to a tournament.
 -- @param client Nakama client.
--- @param tournament_id_str (string) The tournament ID to write the record for..
--- @param body_write_tournament_record_request_tournament_record_write (table) Record input..
+-- @param tournament_id_str (string) The tournament ID to write the record for.
+-- @param score (string) The score value to submit.
+-- @param subscore (string) An optional secondary value.
+-- @param metadata (string) A JSON object of additional properties (optional).
+-- @param operator () Operator override.
+
 -- @param callback Optional callback function.
 -- A coroutine is used and the result returned if no function is provided.
 -- @return The result.
-function M.write_tournament_record2(
-	client
-	,tournament_id_str
-	,body_write_tournament_record_request_tournament_record_write
-	,callback)
+function M.write_tournament_record2(client, tournament_id_str, operator, score, subscore, metadata,callback)
```

Comment:
Same comment as other `"2"` functions. :)

Reply:
Apologies yes I can see they're defined that way to differentiate between POST and GET requests. Let's leave them for now, we can revisit them separately. The Unity client for example internally calls RpcFunc and RpcFunc2 depending on if the call to `RpcAsync` contains a body payload or not. We can look to do something like this in the future for Defold.